src/general/MainMenu.cs

```diff
@@ -340,6 +345,16 @@ private void FreebuildEditorPressed()
         }, false);
     }
 
+    private void OnAutoEvoExploringPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+
+        autoEvoExploringButton.Disabled = true;
+
+        TransitionManager.Instance.AddSequence(ScreenFade.FadeType.FadeOut, 0.1f,
+            () => { SceneManager.Instance.SwitchToScene("res://src/auto-evo/AutoEvoExploringTool.tscn"); }, false);
```

Comment:
I think the exploring tool is a kind of a separate thing, so at least to me it makes a bit more sense to have it separately from the main menu, after all entering the freebuild editor is kind of the same thing of jumping to a different game state. And also I think it'll make the main menu a fractionally lighter scene to not have to put the auto-evo tool in there, which I am in favour of because it's not really something I think most players would find that useful.

Reply:
Yeah, and with this code I don't need to manually clean exploring tool when it exits.

Reply:
Actually related to a recent bug: https://community.revolutionarygamesstudio.com/t/error-when-trying-to-load-save/5139/16 could you make it so that if an auto-evo run is in progress when the auto-evo tool exits the tree (put code in tree exit callback) the auto-evo run is aborted automatically?

Reply:
Sure.